packages/manager/apps/dedicated/client/app/app.js

```diff
@@ -234,7 +234,6 @@ angular
       });
 
       $state.defaultErrorHandler((error) => {
-        console.log(error);
```

Comment:
i don't think it's bad to log errors, otherwise they would remain silent in that case
although it's preferable to use ```console.error```

Reply:
I added this to see the error before and forgot to remove.  Here I removed it.

Reply:
I agree with you, we should have console.error